packages/media/src/test/getFileExtFromUrl.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import getFileExtFromUrl from '../getFileExtFromUrl';
+
+describe('getFileExtFromUrl', () => {
+  it('should find ext', () => {
```

Comment:
Nit: could probably use `it.each` with a few more inputs & expected outputs

Reply:
Improved test https://github.com/google/web-stories-wp/pull/9031/commits/8e93ef0b84587d64ad46cdfc03f711ee48943a43